indexer/indexer.go

```diff
@@ -233,6 +236,15 @@ func ProcessDataFeed(csConsumer transports.Consumer, txFeed *txfeed.TxFeed, db *
 				megaStatement := []string{}
 				megaParameters := []interface{}{}
 				for _, pb := range chainUpdate.Added() {
+					for _, v := range pb.Values {
```

Comment:
This loop is not needed. It's setting safeNum and finalizedNum once for every value in pb.Values, but we only need to get them once.